src/index.tsx

```diff
@@ -30,15 +32,28 @@ Sentry.init({
   dsn: REACT_APP_SENTRY_DSN
 });
 
+const client = new ApolloClient({
```

Comment:
This config shouldn't be a part of index.tsx
Can you make new file under config maybe? and the URI should come from ENV